frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx

```diff
@@ -97,25 +89,32 @@ const ListItem: FC<
     </ListItemRow>
 );
 
+const useProjectResources = (projectId: string) => {
```

Comment:
Is that a common practice we do, having a subhook? I would have used project status hook directly.

Reply:
I ... don't know? I think I've seen Mateusz do it a couple times, so .. yeah? In this case, I just thought it made the main component more legible. But I'm not married to it 💁🏼